app/Http/Controllers/TaskController.php

```diff
@@ -45,7 +68,16 @@ public function store(Request $request)
      */
     public function show($id)
     {
-        //
+        $arr = [];
+        $tasks = DB::table('task_user')->select('task_id')->where('user_id', $id)->get();
+        foreach ($tasks as $item) {
+            $arr[] = $item->task_id;
+        }
```

Comment:
```suggestion
        $tasks->pluck('task_id')
```
chỗ này dùng pluck nhé, 1 hàm của collection để lấy data, ko nên dùng for